### ISSUE TITLE
Label and images export to Firebase gets error

### ISSUE THREAD

**Issue by eugeneyu** [firebase/mlkit-custom-image-classifier#10]
I can create dataset on the app, and create label and take photos locally. But when clicking menu "Export to Firebase", the bottom shows "Error while exporting dataset to Firebase". 

Another issue is on Dataset screen all the listed datasets keep showing "Loading..." instead of their names.

I'm wondering if this is related to the project resource region setting. My Firebase project is set to the location of asia-northeast1, so some resources are created in that region, but the AutoML related bucket etc. are still at us-central1. And the Firebase config files are pointing to us-central1 as well. Could this be an issue?

**Comment by prakhar1989** [comment]
yup, sounds like a location issue.  Can you try with setting the project resource setting to `us-central-1` or try changing this constant to match your region? https://github.com/firebase/mlkit-custom-image-classifier/blob/master/flutter-app/lib/constants.dart#L46
